diff --git a/core/vm/contracts.go b/core/vm/contracts.go
index 7b8757dc2..4c01c1434 100644
--- a/core/vm/contracts.go
+++ b/core/vm/contracts.go
@@ -128,9 +128,6 @@ var PrecompiledContractsCancun = map[common.Address]PrecompiledContract{
 	common.BytesToAddress([]byte{8}):    &bn256PairingIstanbul{},
 	common.BytesToAddress([]byte{9}):    &blake2F{},
 	common.BytesToAddress([]byte{0x0a}): &kzgPointEvaluation{},
-
-	common.BytesToAddress([]byte{102}): &blsSignatureVerify{},
-	common.BytesToAddress([]byte{103}): &cometBFTLightBlockValidate{},
 }

 // PrecompiledContractsHaber contains the default set of pre-compiled Ethereum
--
2.33.0

